### PR TITLE
docker: replace celestia with celestia-da

### LINKED ISSUE #74: [Feature Request]: support celestia-da
### Implementation ideas

For projects integrating with local celestia devnet, it would be great to have `celestia-da` embedded into the docker image, so that integrations don't have to run another process for the gRPC DA service.

### PR DESCRIPTION
**Pull request by tuxcanfly**
## Overview

Fixes #74 

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

## Summary by CodeRabbit

- **New Features**
  - Introduced gRPC service on port 26650 for the Data Availability Service.
  - Updated Docker base image to enhance application stability and performance.

- **Documentation**
  - Updated README.md with new port mappings and environment variable usage instructions.

- **Refactor**
  - Modified entrypoint script to initialize and start the `celestia-da` bridge with new parameters for improved service integration.

- **Chores**
  - Adjusted Dockerfile to expose additional port and include necessary package installations.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM ghcr.io/celestiaorg/celestia-app:v1.3.0 AS celestia-app
 
-FROM ghcr.io/celestiaorg/celestia-node:v0.12.0
+FROM opcelestia/celestia-da:v0.12.0
```
Comment (gupadhyaya) [inline Dockerfile:3]: where is this hosted?

Comment (tuxcanfly) [reply Dockerfile:3]: This will be replaced with `ghcr.io/celestiaorg/celestia-da:v0.12.0`

Comment (tuxcanfly) [reply Dockerfile:3]: New release of `celestia-da` should trigger this after: https://github.com/rollkit/celestia-da/pull/20 is merged

**Comment by gupadhyaya** [comment]
@tuxcanfly can you update the docker file?

**Comment by tuxcanfly** [comment]
OK, updated to use `celestia-da v0.12.1-rc0`. This version bundles `celestia-node dd3b8e1411de`.

**Review by Manav-Aggarwal — DISMISSED** [review]

**Comment by tuxcanfly** [comment]
Namespace can be configured from the environment using `CELESTIA_NAMESPACE` variable.

**Review by Manav-Aggarwal — APPROVED** [review]

**Review by nashqueue — APPROVED** [review]
utAck

**Pull request merged**